### PR TITLE
Theme updates oct17

### PR DESCRIPTION
**Pull request by phil-evans**
- misc framework updates & bug fixes
- template items for news & learn pages

### CODE REVIEW

**Comment by perronld** [comment]
For the future, do you want me to remove framework.css and global.css from the repo, and add them to the sass watcher in the docker compose?

**Comment by perronld** [comment]
Except for my little questions, I approve the PR. Let me know if you want me to have the css removed from the repo.

Thanks!

File: framework/resources/functions/builder/field-groups/block/content/query.php [hunk 1]
```diff
@@ -42,8 +42,8 @@
 						?>
 							
 						<div class="form-check form-check-inline">
-							<input class="form-check-input" type="checkbox" value="<?php echo $post_type; ?>" name="inputs-args-post_type-<?php echo $post_type; ?>" id="inputs-args-post_type-<?php echo $post_type; ?>">
-							<label class="form-check-label" for="inputs-args-post_type-<?php echo $post_type; ?>">
+							<input class="form-check-input" type="checkbox" value="<?php echo $post_type; ?>" name="inputs-args-post_type-<?php echo str_replace ( '-', '*', $post_type ); ?>" id="inputs-args-post_type-<?php echo str_replace ( '-', '*', $post_type ); ?>">
```
[the remark this answers is not in the document]
Comment (phil-evans) [reply framework/resources/functions/builder/field-groups/block/content/query.php:45]: nope - the builder app separates form input names by the `-` characters to build out the JSON object i.e. `inputs-id="element"` becomes `{ inputs: { id: "element" } }`

there are rare occasions where the name contains a `-` that shouldn't be split i.e. `inputs-args-post_type-training-resource`, so it's temporarily replaced with a `*`

**Comment by phil-evans** [comment]
> For the future, do you want me to remove framework.css and global.css from the repo, and add them to the sass watcher in the docker compose?

sure - i'll add it to .gitignore and remove it from tracking

File: framework/resources/functions/builder/ajax.php [hunk 2]
```diff
@@ -149,6 +150,28 @@ function fw_get_builder_object() {
...
+	}
+	
+	$builder = get_post_meta ( $source, 'builder', true );
+	$builder = str_replace ( $source, $target, $builder );
+	
+	// update_post_meta ( $target, 'builder', $builder );
+	
+	return 'success';
+	
+	wp_die();
```
[the remark this answers is not in the document]
Comment (phil-evans) [reply framework/resources/functions/builder/ajax.php:170]: that function is actually not in use - removed entirely in the next commit

**Review by francisPLT — APPROVED** [review]

**Pull request merged**